R/integration.R

```diff
@@ -39,6 +40,46 @@ runSeurat = function(data, batch, hvg=2000) {
         	   verbose = T)
 	  return(integrated)
 }
+
+runSeuratRPCA = function(data, batch, hvg=2000) {
+	  require(Seurat)
+	  batch_list = SplitObject(data, split.by = batch)
+
+          features <- SelectIntegrationFeatures(batch_list)
```

Comment:
this we want to avoid. So can we just put `features <- hvg` or something here for the time being?

Reply:
ok, I'll try that. The current setupd doesn't work either way